src/lib.rs

```diff
@@ -339,13 +379,25 @@ where
         self
     }
 
+    /// Provide a function that print the header
```

Comment:
This needs more explanation with an example output or something. At this point I don't even know what header really means. :D